src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml

```diff
@@ -756,7 +757,12 @@ well as menu structures (for main menu and popup menus).
         desc="Create a new R Markdown document"
         rebindable="false"/>
         
-    <cmd id="newRHTMLDoc"
+   <cmd id="newRShinyApp"
+        menuLabel="R Shiny _Web App..."
```

Comment:
The "R" prefix looks more awkward than I thought it would, let's just make this "Shiny Web App..."